core/src/main/scala/kafka/log/ProducerStateManager.scala

```diff
@@ -846,6 +881,15 @@ case class SnapshotFile private[log] (private var _file: File,
   def file: File = {
     _file
   }
+
+  def renameTo(oldSuffix: String, newSuffix: String): Unit = {
```

Comment:
`oldSuffix` is always passed as empty string by the caller. Do we need the parameter here?